### PR TITLE
feat(SetOrg): update url to match page title

### LINKED ISSUE #5396: UserWidget - Update urls to match navigation titles
**Proposal:**
In the UI, Organization -> Members goes to /orgs/:orgId/users and Organization -> Settings goes to /orgs/:orgId/about. We should have the url match the navigation title (i.e. Members goes to /orgs/:orgId/members, and Settings goes to /orgs/:orgId/settings) for the sake of consistency. 

**Current behavior:**
Organization -> Members goes to /orgs/:orgId/users
Screen Shot 2022-08-15 at 12 19 26 PM

Organization -> Settings goes to /orgs/:orgId/about. 
Screen Shot 2022-08-15 at 12 18 06 PM

**Desired behavior:**
Organization -> Members goes to /orgs/:orgId/members
Organization -> Settings goes to /orgs/:orgId/settings

**Alternatives considered:**
None

**Use case:**
Keeps the code consistent and easier to follow.

### PR DESCRIPTION
**Pull request by blegesse-git**
Closes #5396 

Inside organization page, navigating to Settings tab takes user to `/orgs/:orgId/org-settings` 
navigating to Members tab takes user to `/orgs/:orgId/members`

Pr adds new paths `/org-settings` and `/members` inside `SetOrg.tsx` and updates links to point to the newly added routes. 
For now, old routes `/about` and `/users` will remain in the codebase to maintain backwards compatibility. 

https://user-images.githubusercontent.com/66275100/184909555-8ad967bd-5d32-46b7-84f0-a057e222c470.mov

### Checklist

Authors and Reviewer(s), please verify the following:

- [x] A PR description, regardless of the triviality of this change, that communicates the value of this PR
- [x] [Well-formatted conventional commit messages](https://www.conventionalcommits.org/en/v1.0.0/) that provide context into the change
- [x] Documentation updated or issue created (provide link to issue/PR)
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)
- [ ] Feature flagged, if applicable

### CODE REVIEW

File: cypress/e2e/cloud/users.test.ts
Comment (hoorayimhelping) [inline cypress/e2e/cloud/users.test.ts:11]: do you think this file should be renamed to `members.test.ts`?

File: src/shared/constants/index.ts
Comment (hoorayimhelping) [inline src/shared/constants/index.ts:110]: doesn't look like this variable is being used anywhere other than this file. I say just remove this whole line

File: src/shared/containers/SetOrg.tsx
Comment (hoorayimhelping) [inline src/shared/containers/SetOrg.tsx:261]: can you add a comment here linking to the issue explaining why we have multiple paths?

File: src/shared/containers/SetOrg.tsx
Comment (hoorayimhelping) [inline src/shared/containers/SetOrg.tsx:276]: same suggestion here about using a comment linking to an issue

File: src/utils/deepLinks.ts
Comment (hoorayimhelping) [inline src/utils/deepLinks.ts:4]: let's keep the `/me/about` and `/me/users` deeplinks around for backwards link compatibility, but have them link to the new link. Then let's remove them when the old routes are removed from `SetOrg.tsx`

```ts
'/me/about': `/orgs/${orgId}/org-settings`
...
'/me/users': `/orgs/${orgId}/members`
```

**Review by hoorayimhelping — COMMENTED** [review]
Good work! Have a couple of minor suggestions. Let me know what you think

File: cypress/e2e/cloud/users.test.ts
Comment (blegesse-git) [reply cypress/e2e/cloud/users.test.ts:11]: yes it would make sense to name it `memebers.test.ts` but doing so will require us to rename many files inside `Src/Users` folder, wouldn't it? I fell into this rabbit hole of having to make a lot of changes to many files yesteday because all files linked to `/user` have `user` in the file name

Comment (blegesse-git) [reply cypress/e2e/cloud/users.test.ts:11]: i think this one is your call, bucky. It can be done but it will affect many many files. Same thing with `about.test.ts` unfortunately.

File: src/shared/constants/index.ts
Comment (blegesse-git) [reply src/shared/constants/index.ts:110]: yeah i checked and it's not being used anywhere. Removing it altogether is a good idea

File: cypress/e2e/cloud/users.test.ts
Comment (blegesse-git) [reply cypress/e2e/cloud/users.test.ts:11]: maybe I should write up a ticket to update the folders I mentioned above?

Comment (hoorayimhelping) [reply cypress/e2e/cloud/users.test.ts:11]: great answer! How about you write up a ticket to change all the instances, but we don't worry about it now. And you and I both file this lesson away as a reminder that figuring out the naming of things early on is way easier than renaming things afterwards.

File: src/utils/deepLinks.ts
Comment (hoorayimhelping) [inline src/utils/deepLinks.ts:4]:
```suggestion
  '/me/about': `/orgs/${orgId}/org-settings`,
```

File: src/utils/deepLinks.ts
Comment (hoorayimhelping) [inline src/utils/deepLinks.ts:34]:
```suggestion
  '/me/users': `/orgs/${orgId}/members`,
```

**Review by hoorayimhelping — APPROVED** [review]
LGTM except for the backwards compatibility piece in deepLinks.

We want the existing deep links to point to the new url, not the old one. This is basically a deprecation notice: Any existing deep links out in the wild (like on a documentation page) won't break this way, and it'll give us time to change any links that need to get changed, but we'll eventually switch over to the new deep link.

I'll approve, but make sure to change that before merging

File: src/utils/deepLinks.ts
Comment (blegesse-git) [reply src/utils/deepLinks.ts:4]: @hoorayimhelping both /about and /org-settings are present here. I thought we agreed to keep both urls for the sake of maintaining backwards compatibility, no?

Comment (blegesse-git) [reply src/utils/deepLinks.ts:4]: oh just saw your explanation down below.

File: cypress/e2e/cloud/users.test.ts
Comment (blegesse-git) [reply cypress/e2e/cloud/users.test.ts:11]: [here](https://github.com/influxdata/ui/issues/5428) is the ticket just for reference

**Review by hoorayimhelping — APPROVED** [review]

**Pull request merged**
